apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -462,15 +445,15 @@ const SettingsIconButton = (props: TouchableOpacityProps) => {
 
 const HeaderRightHistory = React.memo(() => {
   const wallet = useSelectedWallet()
-  const {navigateToSettings} = useWalletNavigation()
+  const { navigateToSettings } = useWalletNavigation()
   const navigation = useNavigation<TxHistoryRouteNavigation>()
 
   return (
     <Row>
       {!wallet.isReadOnly && (
         <>
           <CodeScannerButton
-            onPress={() => navigation.navigate('scan-start', {insideFeature: 'scan'})}
+            onPress={() => navigation.navigate('scan-start', { insideFeature: 'scan' })}
             color={COLORS.ACTION_GRAY}
```

Comment:
all colors should comes from the hook `useStyles` , which should have the theme and hook in it: `useTheme`